app/javascript/app/document-capture/components/file-input.jsx

```diff
@@ -55,14 +84,20 @@ function FileInput({ label, hint, bannerText, accept, value, onChange, className
         <div className="usa-file-input__target">
           {value && !isMobile && (
             <div className="usa-file-input__preview-heading">
-              <span className="usa-sr-only">{t('doc_auth.forms.selected_file')}: </span>
-              {value.name}{' '}
+              <span>
+                {value.name && (
+                  <>
+                    <span className="usa-sr-only">{t('doc_auth.forms.selected_file')}: </span>
+                    {value.name}{' '}
```

Comment:
QQ: what's the extra space for? To make sure it's not an empty tag? Would `value.name || ' '` also do the same thing?

Reply:
> QQ: what's the extra space for? To make sure it's not an empty tag? Would `value.name || ' '` also do the same thing?

The problem can be seen in the test which fails if you remove the space:

```
      AssertionError: expected 'doc_auth.forms.selected_file: demo.pngdoc_auth.forms.change_file' to equal 'doc_auth.forms.selected_file: demo.png doc_auth.forms.change_file'
      + expected - actual

      -doc_auth.forms.selected_file: demo.pngdoc_auth.forms.change_file
      +doc_auth.forms.selected_file: demo.png doc_auth.forms.change_file
```

Which is: Without the space, the effective text content of the header (its `textContent`) would run together, since the all of these elements are inline. In practical terms, I'm not sure if this has a negative impact which could affect the user, since at least in my testing of macOS VoiceOver, a screen reader may read the contents of each element individually.